aexpect/ops_windows.py

```diff
@@ -503,6 +503,7 @@ def find_unused_port(session, start_port=10024):
     :returns: unused port that was found
     :rtype: int
     """
+    # pylint: disable=C0301
```

Comment:
I wonder here (and the commit message adding these lines doesn't answer it) why are these changes now? Is it because the CI didn't run for a while? At first I thought it is related to bumping some new version of linter but pydocstyle doesn't seem related here. Even if it was an offline CI (we could clarify this in the commit message) I still don't quite understand how these changes would pass undetected when the pull request was merged (I think the CI worked fine there).

Reply:
We are using the latest python3, which happens to be 3.12 now and it does not work with the old bumped pylint. So yes,I had to update the pylint and related modules and these seem to complain about this line (although it should have been probably detected earlier)

Reply:
Ok, then let's prioritize restoring the CI. On my side I believe it is useful to investigate such external changes and document them in commit messages since pylint is not supposed to change its requirements on pre-existing and already linted issues like C0301.

Reply:
It shouldn't be a new issue, it's just issue that wasn't caught by previous version (probably bugfix on pylint side). But yeah, I could have improved the commit message but as it was just a style fix.